src/openloco/localisation/string_ids.h

```diff
@@ -6,6 +6,8 @@
 
 namespace openloco::string_ids
 {
+    constexpr string_id empty = 0;
```

Comment:
We already have `string_ids::null` through stringmgr, right?

Reply:
`null` is `0xFFFF` or `-1`, making it different from `empty`, which is `0`